taxonomies/ingredients.txt

```diff
@@ -66878,8 +66969,15 @@ de:pencillium roqueforti (Edelschimmel)
 es:Penicillium roqueforti
 fr:penicillium roqueforti
 la:pencillium roqueforti
+pl:pencillium roqueforti
 # ingredient/fr:penicillium-roqueforti has 103 products @2019-01-30
 
+<en:penicillium
+en:penicillium glaucum
+fr:penicillium glaucum
+la:penicillium glaucum
+pl:penicillium glaucum
+
```

Comment:
Better replace this with one xx: line under the en: line, then the other languages lines are no longer necessary.